lib/reek/ast/sexp_extensions.rb

```diff
@@ -169,9 +169,9 @@ def body
 
       # Utility methods for :send nodes.
       module SendNode
-        def receiver() self[1] end
-        def method_name() self[2] end
-        def args() self[3..-1] end
+        def receiver; children.first; end
+        def method_name() children[1]; end
```

Comment:
Nice and clean. We should do something similar in other places as well in the future.